pages/playback/index.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+  <v-col>
+    <swiper
+      v-if="currentQueue"
+      ref="playbackSwiper"
+      class="d-flex justify-center align-center"
+      :options="swiperOptions"
+      @slideChange="onSlideChange"
+    >
+      <swiper-slide
+        v-for="item in currentQueue"
+        :key="item.Id"
+        class="swiper-no-swiping"
```

Comment:
Why disable touch events?

Reply:
@Camc314 Messes with centering when queues have only 1 or 2 items, until we don't have that fixed it's better to leave it disabled imo.